CODE_JSON_GENERATORS.md

```diff
@@ -26,12 +29,45 @@ From the working group session we have captured business needs,  system requirem
 
 Please see the full list here: [https://github.com/GSA/code-gov/projects/1](https://github.com/GSA/code-gov/projects/1)
 
+## GSA Code Inventory Tool
+
+The [Code Inventory](https://github.com/GSA/codeinventory) tool is a Ruby gem application that provides basic functionality in a CLI and API, and it can be extended via plug-ins (which are also gems) to support any type of repository
+One plug-in we developed is to add GitHub support: [https://github.com/GSA/codeinventory-github
+]()
+
+[https://github.com/GSA/code-clerk](New Code Clerk tool)  

```

Comment:
Wrong link formatting. It should be `[New Code Clerk tool](https://github.com/GSA/code-clerk)`